src/util-frontend.js

```diff
@@ -396,20 +396,20 @@ export function timezoneList() {
             })
         } catch (e) {
             console.error(e.message);
-            console.log("Skip this timezone")
+            console.log('Skip this timezone')
         }
 
     }
 
     result.sort((a, b) => {
         if (a.time > b.time) {
             return 1;
-        } else if (b.time > a.time) {
+        } if (b.time > a.time) {
```

Comment:
Look strange to me.

Reply:
Sorry, I forget to add new line

The case: Don't use `else` if we `return` in previous `IF` statement